orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -111,6 +111,8 @@ def __init__(self, module):
             self._torch_alloc = self._torch_gpu_allocator.gpu_caching_allocator_raw_alloc_address()
             self._torch_free = self._torch_gpu_allocator.gpu_caching_allocator_raw_delete_address()
 
+        _utils._load_aten_op_executor_cpp_extension(self._loglevel < _logger.LogLevel.WARNING, self.is_rocm_pytorch)
```

Comment:
let's add a hard-coded flag to block this from using in the common exec path. 
We will move this load process to after getting the onnx exported graph 

self._use_aten_ops = False
if self._use_aten_ops: 
 ,,,,

Reply:
I moved this call to after export and call it only when there is ATenOp in the graph.

Reply:
nice!